core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/add-repositories-noarch.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash -eu
+
+DOCKER_CE_PATCHED_REPO_CONF=$(cat <<'EOF'
+[docker-ce-stable-patched]
+name=Docker CE Stable - patched centos/7/$basearch/stable
+baseurl=https://download.docker.com/linux/centos/7/$basearch/stable
```

Comment:
AFAICU this is not "noarch".

Reply:
Terminology wise no, however I would consider repos that support multiple arches in noarch using the $basearch flag. For me at least it does not make sense to define twice inside:

- add-repositories-aarch64.sh 
- add-repositories-x64_86.sh 

Maybe we should name add-repositories-noaarch64.sh differently? @to-bar  probably has some good wisdom about this:)

Reply:
Yeah, that is not functionality, but rather code organisation. My feeling was that I would separate `add-repositories-noaarch64.sh` file into two: `add-repositories-noaarch64.sh` and `add-repositories-osdynamic.sh` or just `add-repositories-dynamic.sh`, but this is just a suggestion.

Reply:
With that you will need todo the same for x86 up to 5 files which IMO is just excessive for what this is trying to achieve.

Reply:
Maybe some different name for add-repositories-noarch.sh? So it indicates its for  norarch + multi arch repos.

Reply:
Imo A would be nicest as not to overdo the splitting IMO.

Reply:
A sounds good 👍